### PR TITLE
Merge 2 functions together in applications MobileNetV2

### PR DESCRIPTION
**Pull request by bonlime**
_inverted_res_block() and _first_inverted_res_block() are nearly the same. Merged them into one function
1. skip "Expand" part for 0 block
2. made layers names similar to original TF graph names

### CODE REVIEW

**Comment by fchollet** [comment]
Thanks for the PR.

@taehoonlee would you have the bandwidth to review?

File: keras/applications/mobilenetv2.py [hunk 1]
```diff
@@ -454,73 +448,39 @@ def MobileNetV2(input_shape=None,
...
-
-    x = Conv2D(expansion * in_channels, kernel_size=1, padding='same',
-               use_bias=False, activation=None,
-               name='mobl%d_conv_expand' % block_id)(inputs)
-    x = BatchNormalization(epsilon=1e-3, momentum=0.999,
-                           name='bn%d_conv_bn_expand' %
-                           block_id)(x)
-    x = Activation(relu6, name='conv_%d_relu' % block_id)(x)
+    x = inputs
+    prefix = 'mobl_expanded_conv_{}_'.format(block_id)
```
Comment (taehoonlee) [inline keras/applications/mobilenetv2.py]: It seems better to change it more concisely. How about `'block_{}_'.format(block_id)`?

**Comment by taehoonlee** [comment]
@fchollet, @bonlime, This PR can produce the same results as now. LGTM.

**Comment by bonlime** [comment]
@taehoonlee made prefix shorter

**Pull request merged**